docs/MANUAL.md

```diff
@@ -687,6 +694,40 @@ One exception is services frameworks like [Anope](https://github.com/anope/anope
 
 If you're looking for a bot that supports modern IRCv3 features, check out [bitbot](https://github.com/jesopo/bitbot/)!
 
+## Kiwi
+
+[Kiwi IRC](https://github.com/kiwiirc/kiwiirc/) is a web-based IRC client at the bleeding edge of IRCv3 support. In particular, it is the only major client to support fully Oragono's server-side history features. For a demonstration of these features, see the [Oragono testnet](https://testnet.oragono.io/kiwi).
```

Comment:
"at the bleeding edge of IRCv3 support" - we could maybe be a touch less advertisey here, it comes off a bit weird. I might suggest just going with a simple "that supports many IRCv3 features" or "that supports many IRCv3 features and is fairly simple to self-host"